src/parser.ts

```diff
@@ -5528,7 +5533,7 @@ function parseSpreadOrRestElement(
     );
 
     const { token, tokenPos, linePos, colPos } = parser;
-
+    // @ts-expect-error BUG?
     if (token === Token.Assign && token !== closingToken && token !== Token.Comma) {
```

Comment:
Is this a mistake?

Reply:
line 5485 `if (token !== Token.Assign && token !== closingToken && token !== Token.Comma) {`
That's more like it.

I tested this line with `if (token === Token.Assign) {`, all tests still pass. Please fix the code. Thx!

Reply:
But I use `if (token === Token.Assign)`, test also pass.

Reply:
#271